modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/akula.dm

```diff
@@ -77,7 +77,31 @@
 	return markings
 
 /datum/species/akula/get_species_description()
-	return placeholder_description
+	return {"The Azuleans, also widely known as Akula, are one of several sapient species of humanoids and make up one of four primary \"nations\" in current-day charted space. \
+	Their physiology is closely shark-like, yet posesses mammalian features that allow them to exist in both water and land without complication. \
+	They have a semi-rigid skeleton composed of tough and flexible cartilage and a nearly-identical organ structure to that of humans with the addition of gills and a buoyant liver. \
+	Among their most noticeable features is the abundant presence of small hydrodynamic feathers on their heads, looking exactly like hair at a first glance - and their webbed toes and fingers. \
+	All of these are often not completely granted, as Akula are close with genetical modification."}
 
 /datum/species/akula/get_species_lore()
-	return list(placeholder_lore)
+	return list({"The Kingdom of Agurkrral is the preeminent nation of the Azuleans, otherwise known as the Akulans. On the surface, Agurkrral appears to be a land of rigid if not stagnant traditionalists. However, it is a dynamic and relatively progressive nation fueled by a rigorous education system, structured reforms program, and polarizing open debates, using what it perceives as merit as a measure of advancement for all those that dwell within its borders. While appearing backwards initially to some in the Sol Federation may consider to be backwards, they enjoy a high degree of personal freedom and have outdone the Sol Federation when it comes to the creation of an arguably liberal and stable, multi-species empire, allowing any alien species to become a citizen while rigorously upholding the values of free speech, free press, free religion, and the right to life.
+	
+	However, many critics see the country as dangerously nationalist and socialist, despite the Akulans having no concept of socialism due to perceiving no difference between the government and its citizenry, as they offer free universal healthcare, free education, and a universal basic income for those that have earned citizenship through military or government service. More alarmingly, the Akulans openly practice eugenics, leveraging their expertise in genetics and their ability to bioengineer on a massive scale. While other nations have fanatics preaching their religion to convert non-believers, some Akulans pressure those around them to perfect themselves further mentally, physically, and genetically, even targeting species that are naturally shorter to coerce them to make themselves taller. It is said that anyone perceived to have a physical or mental defect are often sent to reformation camps where they undergo intensive and intrusive changes to their genetic makeup with some whispering that the Akulans are secretly conducting a slow genocide.
```

Comment:
make sure you standardize this whitespace before undrafting, currently humanity has none but others have it every other line

Reply:
Might be changed to be similar to Humanity's to be more legible in-game.